Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -129,57 +133,49 @@ - (instancetype)initWithFrame:(CGRect)frame {
  Microsoft Extension
 */
 - (instancetype)initWithFrame:(CGRect)frame xamlElement:(RTObject*)xamlElement {
-    // TODO: We're passing nil to initWithFrame:xamlElement: because we have to *contain* a _subview for padding/layout.
-    // Note: Pass 'xamlElement' instead, once we move to a *single* backing Xaml element for UIActivityIndicatorView.
-    if (self = [super initWithFrame:frame xamlElement:nil]) {
-        [self _initUIActivityIndicatorView:xamlElement];
+    if (self = [super initWithFrame:frame xamlElement:xamlElement]) {
+        [self _initUIActivityIndicatorView];
     }
 
     return self;
 }
 
 /**
  @Status Interoperable
+ @Note Frame will not be resized smaller than the contained ProgressRing, as this would obscure it.
 */
 - (void)setFrame:(CGRect)frame {
-    [super setFrame:frame];
-
-    _subView.center = { self.center.x - self.frame.origin.x, self.center.y - self.frame.origin.y };
-}
-
-- (void)_initUIActivityIndicatorView:(RTObject*)xamlElement {
-    if (xamlElement != nil) {
-        _progressRing = objcwinrt::from_rtobj<Controls::ProgressRing>(xamlElement);
+    if (frame.size.width < [self _width] || frame.size.height < [self _height]) {
+        int updatedFrameHeight = frame.size.height < [self _height] ? [self _height] : frame.size.height;
+        int updatedFrameWidth = frame.size.width < [self _width] ? [self _width] : frame.size.width;
+        frame = CGRectMake(frame.origin.x, frame.origin.y, updatedFrameWidth, updatedFrameHeight);
     }
 
-    if (!_progressRing) {
-        _progressRing = Controls::ProgressRing();
-    }
-
-    // TODO: We should move this over to a single Xaml element which we return from a createXamlElement implementation.
-    //       Which would also mean that we'd just reach into [self xamlElement] to get the backing _progressRing,
-    //       and we would no longer have a _subview.
-    _subView = [[UIView alloc] initWithFrame:CGRectZero
-                                 xamlElement:(xamlElement ? xamlElement : objcwinrt::to_rtobj(_progressRing))];
+    [super setFrame:frame];
+}
 
-    [self addSubview:_subView];
+- (void)_initUIActivityIndicatorView {
     _isAnimating = NO;
+    [self setHidden:NO]; // Ensure space is reserved in the layout. See notes for setHidesWhenStopped.
```

Comment:
isn't NO the default?  if we're never setting to YES, then we don't ever need to set it to NO, right?